Habitica/res/values/strings.xml

```diff
@@ -347,4 +358,12 @@ To start, which parts of your life do you want to improve?</string>
     <string name="open_in_store">Open in play store</string>
     <string name="change_class_confirmation">Are you sure you want to change your class? This will cost 3 gems.</string>
     <string name="leaderMessage" formatted="false">Message from %1$s</string>
+
+    <!-- Local notification actions -->
+    <string name="accept_party_invite">ACCEPT_PARTY_INVITE</string>
+    <string name="reject_party_invite">REJECT_PARTY_INVITE</string>
+    <string name="accept_guild_invite">ACCEPT_GUILD_INVITE</string>
+    <string name="reject_guild_invite">REJECT_GUILD_INVITE</string>
+    <string name="accept_quest_invite">ACCEPT_QUEST_INVITE</string>
+    <string name="reject_quest_invite">REJECT_QUEST_INVITE</string>
```

Comment:
These should be marked with `translatable="false"`, so that they are not accidentally translated.

Maybe they should also be moved to strings.constants.xml?